Makefile

```diff
@@ -268,3 +268,14 @@ generate-kustomize: bin/helm
 	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/serviceaccount-csi-node.yaml | sed -e "/namespace: /d" > ../../deploy/kubernetes/base/serviceaccount-csi-node.yaml
 	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/role-leases.yaml | sed -e "/namespace: /d" > ../../deploy/kubernetes/base/role-leases.yaml
 	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/rolebinding-leases.yaml | sed -e "/namespace: /d" > ../../deploy/kubernetes/base/rolebinding-leases.yaml
+
+.PHONY: update-truth-sidecars
+update-truth-sidecars: hack/release-scripts/image-digests-template.yaml hack/release-scripts/get-latest-sidecar-images
+	./hack/release-scripts/get-latest-sidecar-images
+
+.PHONY: generate-sidecar-tags
+generate-sidecar-tags: hack/release-scripts/image-digests.yaml charts/aws-ebs-csi-driver/values.yaml deploy/kubernetes/overlays/stable/gcr/kustomization.yaml hack/release-scripts/generate-sidecar-tags
+	./hack/release-scripts/generate-sidecar-tags
+
+.PHONY: update-sidecar-dependencies
+update-sidecar-dependencies: update-truth-sidecars generate-sidecar-tags generate-kustomize
```

Comment:
Why does this run `generate-kustomize`?

Reply:
Because `generate-kustomize` will propagate the updates in `deploy/kubernetes/overlays/stable/gcr/kustomization.yaml` to `deploy/kubernetes/base/controller.yaml` and `deploy/kubernetes/base/node.yaml` 

e.g. [csi-provisioner in base/controller.yaml](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/97688f8549feb99228f442af725c0bd492ac9a60/deploy/kubernetes/base/controller.yaml#L131)